modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItSqlMultiStatementTest.java

```diff
@@ -138,131 +126,76 @@ void queryWithIncorrectNumberOfDynamicParametersFailsWithValidationError() {
         assertThrowsSqlException(STMT_VALIDATION_ERR, expectedMessage, () -> runScript(sql, null, 0, 1, 2, 3, 4, 5));
     }
 
-    @Test
-    void transactionControlStatementDoesNotCreateCursor() {
-        assertThat(runScript("START TRANSACTION; COMMIT"), nullValue());
-
-        AsyncSqlCursor<List<Object>> cursor = runScript(
-                "START TRANSACTION;"
-                        + "SELECT 1;"
-                        + "COMMIT"
-        );
-
-        assertNotNull(cursor);
-        validateSingleResult(cursor, 1);
-
-        assertFalse(cursor.hasNextResult());
-    }
-
     @Test
     void scriptStopsExecutionOnError() {
         // Runtime error.
-        AsyncSqlCursor<List<Object>> cursor = runScript(
-                "CREATE TABLE test (id INT PRIMARY KEY);"
-                + "SELECT 2/0;" // Runtime error.
-                + "INSERT INTO test VALUES (0)"
-        );
-        assertNotNull(cursor);
-        assertTrue(cursor.hasNextResult());
+        {
+            AsyncSqlCursor<List<Object>> cursor = runScript(
+                    "CREATE TABLE test (id INT PRIMARY KEY);"
+                            + "SELECT 2/0;" // Runtime error.
+                            + "INSERT INTO test VALUES (0)"
+            );
+            assertNotNull(cursor);
```

Comment:
we already have notNull check inside BaseSqlMultiStatementTest#runScript() i suppose this check is redundant. Also do we need to check here if empty table is created ?

Reply:
Thanks. Removed all redundant null-checks that I found